cmd/template/cluster/flag.go

```diff
@@ -78,6 +84,33 @@ func (f *flag) Init(cmd *cobra.Command) {
 func (f *flag) Validate() error {
 	var err error
 
+	if f.ClusterID != "" {
+		if len(f.ClusterID) != key.IDLength {
+			return microerror.Maskf(invalidFlagError, "--%s must be length of %d", flagClusterID, key.IDLength)
+		}
+
+		if _, err := strconv.Atoi(f.ClusterID); err == nil {
```

Comment:
I think it would be better to stick with a regex here because (I think--haven't tested) "01234" would throw an error due to the leading 0 and would therefore be accepted.